scripts/build-engine/package-lock.json

```diff
@@ -259,24 +259,45 @@
       }
     },
     "@babel/generator": {
-      "version": "7.9.4",
-      "resolved": "https://registry.npm.taobao.org/@babel/generator/download/@babel/generator-7.9.4.tgz?cache=0&sync_timestamp=1585038762678&other_urls=https%3A%2F%2Fregistry.npm.taobao.org%2F%40babel%2Fgenerator%2Fdownload%2F%40babel%2Fgenerator-7.9.4.tgz",
-      "integrity": "sha1-EkQekMOzxBWc3s8xIHW/Gozi284=",
-      "requires": {
-        "@babel/types": "^7.9.0",
-        "jsesc": "^2.5.1",
-        "lodash": "^4.17.13",
-        "source-map": "^0.5.0"
```

Comment:
Why need to update this file?

Reply:
I checked with @PPpro, this change will cause compatibility issues, and have rolled back.